deploy/examples/create-external-cluster-resources.py

```diff
@@ -675,15 +654,14 @@ def get_ceph_external_mon_data(self):
                 "Only 'v2' address type is enabled, user should also enable 'v1' type as well"
             )
         ip_port = str(q_leader_details["public_addr"].split("/")[0])
-        return "{}={}".format(str(q_leader_name), ip_port)
+        return f"{str(q_leader_name)}={ip_port}"
 
     def _join_host_port(self, endpoint, port):
-        port = "{}".format(port)
```

Comment:
It should has to typed as string 
` port = "{}".format(port) `  ->   port = str(port)

Currently the CI fail for this error

Reply:
Corrected. Removed this by mistake in 2nd iteration.